vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/ElementCssSteps.java

```diff
@@ -0,0 +1,235 @@
+     * <p>The expected CSS parameters to be defined in the ExamplesTable:</p>
+     * <ul>
+     * <li><b>cssProperty</b> - the name of the CSS property</li>
+     * <li><b>comparisonRule</b> - String comparison rule: "is equal to", "contains", "does not contain",
+     * "matches".</li>
+     * <li><b>expectedValue</b> - expected CSS property value</li>
+     * </ul>
+     * <p>Usage example:</p>
+     * <code>
+     * <br>Then context element does have CSS properties matching rules:
+     * <br>|cssProperty |comparisonRule |expectedValue |
+     * <br>|border      |contains       |solid         |
+     * </code>
+     *
+     * @param parameters The parameters of the expected CSS properties to set as ExamplesTable
+     */
+    @Then("context element does have CSS properties matching rules:$parameters")
+    public void doesElementHasCssProperties(List<CssValidationParameters> parameters)
+    {
+        uiContext.getSearchContext(WebElement.class).ifPresent(element -> {
+            String getAllCssScript =
+                    org.vividus.util.ResourceUtils.loadResource("org/vividus/ui/web/get-element-computed-css-func.js");
+            getAllCssScript += "return getComputedStyleAsMap(arguments[0])";
+            Map<String, String> elementCss = javascriptActions.executeScript(getAllCssScript, element);
+
+            List<CssValidationResult> cssResults = validateElementCss(parameters, elementCss);
+            attachmentPublisher.publishAttachment("templates/css_validation_result.ftl",
+                    Map.of("cssResults", cssResults), "Css validation results");
+        });
+    }
+
+    private List<CssValidationResult> validateElementCss(List<CssValidationParameters> parameters,
+                                                         Map<String, String> elementCss)
+    {
+        List<CssValidationResult> cssResults = new ArrayList<>();
+        parameters.forEach(param ->
+        {
+            String cssName = param.getCssProperty();
+            String expectedValue = param.getExpectedValue();
+            StringComparisonRule comparisonRule = param.getComparisonRule();
+
+            String actualCssValue = getCssValue(elementCss, cssName);
+            boolean passed = softAssert.assertThat(String.format("Element has CSS property '%s' %s value '%s'",
+                            cssName, comparisonRule, expectedValue),
+                    actualCssValue, comparisonRule.createMatcher(expectedValue));
+            cssResults.add(new CssValidationResult(param, expectedValue, passed));
+        });
+        return cssResults;
+    }
+
+    private String getCssValue(Map<String, String> cssMap, String cssName)
+    {
+        return Optional.ofNullable(cssMap.get(cssName)).orElseGet(() -> {
+            String cssValueAsCamelCase = CaseUtils.toCamelCase(StringUtils.removeStart(cssName, '-'), false, '-');
+            return cssMap.get(cssValueAsCamelCase);
+        });
+    }
```

Comment:
it makes sense to introduce `CssValidations` class, move this logic to it and re-use in both places